### ISSUE TITLE
New page: New User Guide (new_user_guide.md)

### ISSUE THREAD

**Issue by badarotti** [openSUSE/openSUSE-docs-revamped#8]
# New page: New User Guide
File: new_user_guide.md
Live link: https://badarotti.github.io/openSUSE-docs-revamped/new_user_guide/

This page is meant to be a quick explainer for new user of the choices they have to make to install openSUSE. It gives them a brief abstract of what the choice is (Ex: Leap Vs. Tumbleweed) then points them to the page where this choice is discussed in details. It is the first page in the Pre-Install documentation section that will explain all those choices. 

Please discuss the page here and if all goes well, I'll make a PR.
Thanks

---
# New User Guide

The goal of the documentation on this site is to get your system up and running as fast as possible with minimum configuration. Like with any operating system you need to make a couple of decisions in relations of your current hardware and what kind of technologies you want to use. Here we will explain the essentials for you to start your installation.

!!! note
    You can install a system that you can use as a Desktop or a Server. This section of the documentation focuses on setting up a desktop system.

   
### Distributions

A distribution is a version of the openSUSE operating system. It contains the  installer, the operating system and the programs necessary to make your computer operational. 

When you go to [get.opensuse.com](https://get.opensuse.org/) to download openSUSE you will find that there are two main distributions. Leap (stable) and Tumbleweed (Rolling).

Pre-Install > [The distribution Guide](distribution.md)

### Installation Media

When you download the version of openSUSE you wish to use, you will download an single image file with a (.iso) extension. You write the ISO image file to a USB flash drive, a blank DVD or CD to create the **Installation Media** that includes the installer and the applications that will make your computer ready for everyday use. Simply boot you computer from the Installation Media to begin the installation process.

Pre-Install > [Installation Media Guide](install_media.md)

### Desktop Environments

Linux gives you lots of choices including the various DE (Desktop Environment) which is the GUI of the operating system. If you're a new Linux user we recommend using the default KDE DE, it is very slick, powerful yet easy to use. If you feel more adventurous you can change the DE during the installation or anytime after you install openSUSE.

Pre-install > [Desktop Environments Guide](desktop_environment.md)

### Drive Partitioning

If you are installing openSUSEon a new drive or you want the openSUSE installer to write over your whole drive deleting everything on it, then the installer will give you that option during installation and you won't have to do any preparations before hand.

In case you you want to preserve an existing partitions and the data it contains, or if you want to dual boot openSUSE with another operating system like Windows; you will find useful tips and tricks in the following guides to achieve your goal safely.

Pre-Install > [Partitioning Guide](drive_partition.md)  
Pre-install > [Dual Booting Guide](dual_boot.md)

### Disk Encryption

Linux has the technology to encrypt your drive for security reasons. If this technology is enabled you will be asked for your passphrase every time you boot your computer. It is recommended to enable disk encryption on laptops as well as desktop computers that are placed in environments that you do not completely control.

Pre-Install > [Disk Encryptio Guide](disk_encryption.md)

### Snapshots & Btrfs

openSUSE uses Btrfs file system and it's snapshot technologies to automatically restore the operating system to a previous known working state in case something goes wrong with an update or upgrade. The technology is automatic and the restoration can be done using the GUI or via the command line interface. 

Pre-install > [Snapshots and Btrfs Guide](btrfs.md)

**Comment by surprized** [comment]
Little typo here at "Pre-isntall considerations", little things aside, I like it. It goes through exactly the same things I would when considering to install openSUSE to someone.
> You can install a system that you can use as a Desktop or a Server. This section of the documentation focuses on setting up a desktop system.
> ## Pre-isntall considerations
> ### Distributions

**Comment by badarotti** [comment]
> Little typo here at "Pre-isntall considerations"
:ok_hand:

**Comment by Nycticorax** [comment]
Awesome, only think I would change is:

**Snapshots & Brts**
I would somehow sort this under a "Pick your filesystem" section. Then I would say that:

- picking Btrfs is almost required for TW, and then refer the user to the relevant docs explaining TW and snapshots
- not picking Btrfs is OK for Leap
- picking Brtfs is recommended for any oS distro given how it synergizes with other tools around which the distros were designed (YaST, snapper, and ultimately, Factory in the sense that Brtfs snapshots work harmoniously with an open-ended pool of packages the user might pull from different third-parties.

**Comment by badarotti** [comment]
> picking Btrfs is almost required for TW, and then refer the user to the relevant docs explaining TW and snapshots

@Nycticorax I never tried TW without Btrfs but I though the installer selects Btrfs or not depending on the root partition size. Or is that just in Leap?

**Comment by Nycticorax** [comment]
> > picking Btrfs is almost required for TW, and then refer the user to the relevant docs explaining TW and snapshots
> 
> @Nycticorax I never tried TW without Btrfs but I though the installer selects Btrfs or not depending on the root partition size. Or is that just in Leap?

TW works fine with ext4, which the user can choose in the installer. I don't know exactly what alrogithm is used behind the curtains by the installer to pre-select something.

**Comment by badarotti** [comment]
I checked the official Leap docs again, It doesn't select Btrfs depending on the root partition size as I though. It selects whether to enable the snapshot capabilities if the partition is bigger than 16G, the partition has to be setup as Btrfs in the first place. The docs also recommend 40G for the root partition.

Ref: https://doc.opensuse.org/documentation/leap/startup/single-html/book-opensuse-startup/index.html#sec-yast-install-partitioning-important

Perhaps we need to spin off a couple of VM to try different scenarios on how the installer recommends things depending on how much space we give it.

**Comment by Nycticorax** [comment]
> Perhaps we need to spin off a couple of VM to try different scenarios on how the installer recommends things depending on how much space we give it.

Nah, I don't think we do need do to that (for now at least). For now I think it suffices to make the following claims:

- as a rule of thumb, it's recommended to pick Btrfs for any oS distro
- Leap? then it's OK to pick whichever filesystem you want.
- TW? then it's NOT OK to pick something else than Brtfs
- when picking Btrfs, make sure you have: 
  - at least 16 GBs available disk space if you want your system to install with snapshots on by default
  - at least 30 GBs available disk space if like above but in addition you want to use snapshots and rollback (40 GBs reocommended though)

**Comment by why-not-try-calmer** [comment]
Not sure about this. Content has changed but we could recycle.
